app/controllers/supervisor/base_controller.rb

```diff
@@ -12,7 +12,7 @@ def must_be_supervisor
   end
 
   def supervised_teams
-    @supervised_teams = current_user.teams.where("roles.name = 'supervisor'")
+    @supervised_teams = current_user.teams.where(:roles => { name: 'supervisor' }, :season_id => Season.current.id)
```

Comment:
This is super picky, but it would be nice to stick to a single Hash syntax (`:a => 10` vs `a: 10`), instead of mixing them. I'd stick with what the rest of the app uses, but in the end it comes down to personal preference whichever of the two styles you use.